decision_maker.go

```diff
@@ -292,13 +292,35 @@ func getTimeout(r *release) string {
 
 // getValuesFiles return partial install/upgrade release command to substitute the -f flag in Helm.
 func getValuesFiles(r *release) string {
+	var fileList []string
+
 	if r.ValuesFile != "" {
-		return " -f " + pwd + "/" + relativeDir + "/" + r.ValuesFile
+		fileList = append(fileList, pwd+"/"+relativeDir+"/"+r.ValuesFile)
 	} else if len(r.ValuesFiles) > 0 {
 		for i := 0; i < len(r.ValuesFiles); i++ {
 			r.ValuesFiles[i] = pwd + "/" + relativeDir + "/" + r.ValuesFiles[i]
 		}
-		return " -f " + strings.Join(r.ValuesFiles, " -f ")
+		fileList = append(fileList, r.ValuesFiles...)
+	}
+
+	if r.SecretFile != "" {
+		if ok := decryptSecret(r.SecretFile); !ok {
```

Comment:
It might be good to add a check before attempting a decrypt to see if helm secrets is available or not (by checking `toolExists("helm secrets help")`.